dbt_subprojects/solana/models/_sector/dex/bot_trades/solana/platforms/bonkbot/bonkbot_solana_base_trades.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    schema = 'bonkbot_solana',
+    alias = 'base_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    incremental_strategy = 'merge',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+    unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index']
```

Comment:
```suggestion
    unique_key = ['block_month', 'tx_id', 'outer_instruction_index', 'inner_instruction_index']
```

Reply:
having the partition forced into merge join conditions should help trino engine focus only on necessary partitions to speed it up. the query planner likely figures it out on its own, but let's be explicit